integration/spark2/src/main/java/org/apache/carbondata/spark/vectorreader/ColumnarVectorWrapper.java

```diff
@@ -25,198 +25,204 @@
 import org.apache.carbondata.spark.util.CarbonScalaUtil;
 
 import org.apache.parquet.column.Encoding;
-import org.apache.spark.sql.execution.vectorized.ColumnVector;
+import org.apache.spark.sql.CarbonVectorProxy;
 import org.apache.spark.sql.types.Decimal;
 
 class ColumnarVectorWrapper implements CarbonColumnVector {
 
-  private ColumnVector columnVector;
+  private CarbonVectorProxy writableColumnVector;
```

Comment:
it is better to name this member a general name instead of a class name in spark2.3

Reply:
agree